packages/commonwealth/server/controllers/server_groups_methods/refresh_membership.ts

```diff
@@ -0,0 +1,40 @@
+import { ChainInstance } from 'server/models/chain';
+import { ServerChainsController } from '../server_chains_controller';
+import { AddressInstance } from '../../models/address';
+import { UserInstance } from '../../models/user';
+
+export type RefreshMembershipOptions = {
+  user: UserInstance;
+  chain: ChainInstance;
+  address: AddressInstance;
+  topicId: number;
```

Comment:
Are we going to refresh for one topic at a time, or are we going to permit refreshing across all topics on a community?

Reply:
Ah yes, I suppose we'd want to refresh across the whole community. I'll remove the topic ID.

Reply:
We can make it optional.